### PR TITLE
fix docker deploy with make undeploy.docker

### PR DESCRIPTION
**Pull request by shalk**
```
make undeploy.docker
```
command have some error.  so i fix it

### CODE REVIEW

**Comment by wu-sheng** [comment]
Thanks for fixing. Related issue https://github.com/apache/skywalking/issues/12077

@wankai123 Could you review and verify?

**Comment by wankai123** [comment]
The others LGTM

File: deploy/platform/docker/docker-compose.elasticsearch-monitor.yaml [hunk 1]
```diff
@@ -21,6 +21,24 @@
 version: '2.1'
 
 services:
+  elasticsearch:
+    image: ${ES_IMAGE}:${ES_IMAGE_TAG}
```
[the remark this answers is not in the document]
Comment (shalk) [reply deploy/platform/docker/docker-compose.elasticsearch-monitor.yaml:25]: done

File: deploy/platform/docker/docker-compose.elasticsearch-monitor.yaml [hunk 2]
```diff
@@ -21,6 +21,24 @@
...
+      retries: 3
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
   es-exporter:
```
[the remark this answers is not in the document]
Comment (shalk) [reply deploy/platform/docker/docker-compose.elasticsearch-monitor.yaml:42]: done

**Review by wankai123 — APPROVED** [review]

**Pull request merged**